src/amo/css/inc/vars.scss

```diff
@@ -14,7 +13,9 @@ $accent-color: #ebebeb;
 $breakpoints: (
   small: '(max-width: 499px)',
   medium: '(min-width: 500px)',
-  large: '(min-width: 720px)'
+  large: '(min-width: 720px)',
+  desktop: only screen and (min-width: 720px),
+  mobile: only screen and (max-width: 719px),
```

Comment:
On reflection I feel that the mix of device agnostic naming with device class naming might be confusing in the future once we have a mix of all of that across the code base. 

Maybe we should pick one, use that and not have a mix?

I'm not tied to the completely agnostic sizes but I think a single system will be better than two. 

My own two cents - I've used more of a mobile/desktop split in the past and what I don't like about that is when you might have a mobile that is actually going to use the desktop breakpoint the concept starts to fall apart. Although, I can buy it a little bit more if the intent is more generic, e.g. mobile and desktop is more a division between 100% width (e.g. mobile layout) and adding additional columns (desktop).

Maybe we only really need a couple of breakpoints though, because I would imagine the rest are going to be specific to the content and would therefore be defined specifically and sit with the related styles - they might actually? I would imagine we're not even using much of these small, medium and large breakpoints yet?

Also since the site is mobile first, there shouldn't be much of a use-case for using the small/mobile breakpoint since that *is* the default and only larger breakpoints should modify it.

Here's some additional food for thought: 

* https://medium.freecodecamp.com/the-100-correct-way-to-do-css-breakpoints-88d6a5ba1862
* https://seesparkbox.com/foundry/there_is_no_breakpoint